android/src/main/java/io/customer/reactnative/sdk/CustomerIOReactNativeInstance.kt

```diff
@@ -27,9 +27,9 @@ object CustomerIOReactNativeInstance {
         val region = environment.getProperty<String>(
             Keys.Environment.REGION
         )?.takeIfNotBlank().toRegion()
-        val organizationId = environment.getProperty<String>(
-            Keys.Environment.ORGANIZATION_ID
-        )?.takeIfNotBlank()
+        val enableInApp = configuration?.getProperty<Boolean>(
+            Keys.Config.ENABLE_IN_APP
+        ) ?: false
```

Comment:
Removing the `organizationId` environment property would remove backwards compatibility from our SDK and therefore, produce a breaking change to the SDK for customers? 

I was expecting to see us keeping the `organizationId` environment property, [deprecating it](https://github.com/customerio/issues/issues/8990#issuecomment-1398753098), and having some code similar to: 

```ts
val organizationId = environment.getProperty<String>(...)
var enableInApp = configuration?.getProperty(...) ?: organizationId != null ?: false
```

Reply:
Ah, now I see this is handled in `src/CustomerioTracking.tsx`. Perfect place for it. Love the solution.